ExamplePlugin/ExamplePlugin.cs

```diff
@@ -43,7 +43,8 @@ public void Awake()
             myItemDef = new ItemDef
             {
                 //More on these later
-                nameToken = "EXAMPLE_CLOAKONKILL_NAME",
+                name = "EXAMPLE_CLOAKONKILL_NAME",
```

Comment:
that will error unless im mistaken, internal names shouldnt have things like ' or spaces

Reply:
Forget that, that should work just fine, I thought for a moment that it would get transformed into its value counterpart